lib/matplotlib/dates.py

```diff
@@ -274,8 +274,7 @@ def _from_ordinalf(x, tz=None):
 class strpdate2num(object):
     """
     Use this class to parse date strings to matplotlib datenums when
-    you know the date format string of the date you are parsing.  See
-    :file:`examples/load_demo.py`.
+    you know the date format string of the date you are parsing.
```

Comment:
the date format string of the date being parsed is known. (May as well remove the second person...?)